symengine/eval_double.cpp

```diff
@@ -197,6 +197,8 @@ class EvalDoubleVisitor : public BaseVisitor<U> {
             result_ = std::atan2(0, -1);
         } else if (eq(x, *E)) {
             result_ = std::exp(1);
+        } else if (eq(x, *EulerGamma)) {
+            result_ = 0.5772156649015328606065; //use until polygamma or digamma is not implemented
```

Comment:
`//use until polygamma or digamma is implemented` ? (Remove `not`, add space after `//`.)